### DISCUSSION TITLE
704

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#883] [Comments]
# 704

블로그의 댓글을 Utterances 에서 Giscus 로 마이그레이션 했다. 기존 댓글을 잘 사용하고 있었지만, 장점이 훨씬 많은 Giscus로 이관하게 되었다. 1. Giscus 장점 Giscus 는 Utterances에 비해 많은 장점들이 있다. 대댓글 댓글 수, 댓글 정렬, 게시물에 대한 반응 다양한 테마 생각보다 많은 테마와 사용자가 직접 생성한 테마를 적용할 수 있다. 블로그에 적용된 테마는 Github Light High Contrast 테마이다. (High Contrast 테마는 JetBrains IDE에서도 사용하는 애정하는 테마이다.) 그 외 장점 지연 로딩 댓글 다국어 (한국어 포함) 봇 자체 호스팅 기타 당등 고급 옵션과 같은 많은 새로운 기능을 추가 가장 중요한건 실제로 계속해서 관..

https://jojoldu.tistory.com/704

**Comment by sjquant** [comment]
오... 댓글창이 많이 예쁘네요.

**Comment by Hong-Mu** [comment]
라벨을 이용하면 빠르게 마이그레이션 가능합니다! [참고링크](https://shipit.dev/posts/from-utterances-to-giscus.html)

**Comment by zhtmr** [comment]
giscus 작동하나요? 404가 뜨네요..

**Comment by dlwlsdud0912** [comment]
안녕하세요 깃허브 첫등록

**Reply by dlwlsdud0912** [reply]
네 어디어디가 맛있어요

**Comment by dlwlsdud0912** [comment]
감사합니다

**Comment by yayyz** [comment]
감사합니다! 덕분에 잘 적용했습니다 :)

**Comment by hi-rachel** [comment]
잘 참고했습니다!
